helpers/getTestimonials.ts

```diff
@@ -1,40 +1,103 @@
-import type { TinaMarkdownContent } from "tinacms/dist/rich-text";
-import client from "../.tina/__generated__/client";
-
-export type TestimonialType = {
-  name: string;
-  avatar?: string;
-  rating: number;
-  company?: string;
-  body?: TinaMarkdownContent;
-};
-
-export const getTestimonialsByCategories = async (
-  categories: string[]
-): Promise<TestimonialType[]> => {
-  const testimonials = await client.queries.testimonalsQuery({
-    categories,
-  });
-
-  let testimonialsResult = testimonials.data.testimonialsConnection.edges.map(
-    (t) => t.node
-  );
-
-  // Adds default/general testimonials if not filled by any testimonial with matching categories
-  if (testimonialsResult.length === 0) {
-    const generalTestimonials = await client.queries.testimonalsQuery({
-      categories: "General",
-    });
-
-    const generalTestimonialsResult =
-      generalTestimonials.data.testimonialsConnection.edges.map((t) => t.node);
-
-    testimonialsResult = generalTestimonialsResult;
-  }
-
-  testimonialsResult = testimonialsResult
-    .sort(() => 0.5 - Math.random())
-    .slice(0, 3);
-
-  return testimonialsResult;
-};
+import testimonialList from "../content/testimonials/testimonials.json";
+import { extractFileName } from "./functions";
+export type TestimonialType = {
+  name: string;
+  avatar?: string;
+  company?: string;
+  body?: string;
+};
+const SSWInternalTestimonialCategories = ["internship", "brainstorming"];
+
+export const getTestimonialsByCategories = async (
+  categories: string[] = [],
+  isOrdered: boolean = true
```

Comment:
i'd recommend not having this 2nd param as:
- it's name isn't that clear (have to look at the code to work out what it does)
- it breaks SRP

instead have a method like `getRandomTestimonialsByCategory` which calls this method under the hood, and then randomises the return value

Reply:
I agree - Fixed